### ISSUE TITLE
sfdx force:source:retrieve -m always returns "No results found" when using comma-separated list

### ISSUE THREAD

**Issue by mauricio-custodio** [forcedotcom/cli#380]
Executing retrieve with comma-separated values is not working anymore
```
> sfdx force:source:retrieve -m ApexPage,ApexClass,ApexTrigger
=== Retrieved Source
No results found
```

And yet this works
```
> sfdx force:source:retrieve -m ApexPage
(...success)

> sfdx force:source:retrieve -m ApexClass
(...success)

> sfdx force:source:retrieve -m ApexTrigger
(...success)
```

### Additional information

**SFDX CLI Version**:
```sfdx-cli/7.53.0 win32-x64 node-v12.16.2```

**SFDX plugin Version**:
```
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.3 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.9 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.7 (core)
analytics 1.7.1 (core)
generator 1.1.2 (core)
salesforcedx 48.8.0 (core)
├─ salesforcedx-templates 48.8.0 (core)
├─ salesforce-alm 48.8.0 (core)
└─ @salesforce/sfdx-plugin-lwc-test 0.1.5 (core)

sfdx-cli 7.53.0 (core)
```

**OS and version**:
Windows 10 Home 
v1909

**Comment by shetzel** [comment]
You're right, this is broken.  I'll make sure we get this in the next patch release.  Thanks for the clear, concise issue!

**Comment by shetzel** [comment]
I'm not positive, but this could just be a shell issue.  If you wrap the comma separated list in quotes it will work.

**Comment by Pyronewbic** [comment]
I've run into this with salesforce-alm at times: Here's the workaround I have (for a mdapi:report:deploy)

```  
                let object = {}
                res = await MdapiDeployReportCommand.run(['-i', this.deploymentId, '-u', this.sharedConn.getUsername(), '--json'])
                for (const record of res.details.componentSuccesses) {
                    if (record.changed == "true" && record.success == "true" && record.fullName != "package.xml") {
                        if (_.has(object, record.componentType)) {
                            object[record.componentType].push(record.fullName)
                        } else if (!_.has(object, record.componentType)) {
                            _.set(object, record.componentType, [])
                            object[record.componentType].push(record.fullName)
                        }
                    }
                }

                this.ux.log(`Retrieving the source for changed and successfully deployed components\n`)
                for (const property in object) {
                    if (object[property].length != 0) {
                        let str = ''
                        for (const val of object[property]) {
                            if (str.length == 0) {
                                str = `${property}:${val}`
                            } else {
                                str = `${str}, ${property}:${val}`
                            }
                        }
                        await SourceRetrieveCommand.run(['-m', str, '-u', this.sharedConn.getUsername()])
                    }
                }
```

**Comment by nramyasri-sf** [comment]
@hammerox , @Pyronewbic  this appears to be fixed. Let me know if you are still running into issues.

**Comment by mshanemc** [comment]
closed as fixed on 8/24/2020 W-7520414
